cassandra3/src/main/java/zipkin2/dependencies/cassandra3/CassandraDependenciesJob.java

```diff
@@ -150,6 +150,9 @@ public CassandraDependenciesJob build() {
     df.setTimeZone(TimeZone.getTimeZone("UTC"));
     this.dateStamp = df.format(new Date(builder.day));
     this.conf = new SparkConf(true).setMaster(builder.sparkMaster).setAppName(getClass().getName());
+    if (builder.sparkMaster.startsWith("local[")) {
+      conf.set("spark.driver.bindAddress", "127.0.0.1");
```

Comment:
this is a spark 3.4 thing

Reply:
basically it tries to detect with the hostname, which isn't needed for local mode anyway.